docs/src/style.md

```diff
@@ -426,8 +424,30 @@ Prefer `using ModuleName: x, p` to `import ModuleName.x, ModuleName.p` and
 `import MyModule: x, p` because the `import` versions allow method extension
 without qualifying with the module name.
 
-Design principles
------------------
+## Documentation
+
+This section describes the writing style that should be used when writing
+documentation for JuMP (and supporting packages).
+
+We recommend the [https://developers.google.com/style/](Google documentation
```

Comment:
I'd recommend adding some additional "general reading" links so that the style guide doesn't seem too biased towards Google. There's been a lot said about writing software documentation.

Reply:
I added a couple. Any other suggestions? Otherwise this is good to merge on my end.